platform/kaniko_cache.go

```diff
@@ -94,6 +95,17 @@ func createKanikoCacheWarmerPod(ctx context.Context, client client.Client, platf
 							MountPath: builder.KanikoCacheDir,
 						},
 					},
+					SecurityContext: &corev1.SecurityContext{
+						AllowPrivilegeEscalation: util.Pbool(false),
+						Privileged:               util.Pbool(false),
+						RunAsNonRoot:             util.Pbool(true),
```

Comment:
Can you please create an alias for these functions within our own `utils` package?